spec/models/asset_spec.rb

```diff
@@ -1033,7 +1034,7 @@
       it "removes the underlying directory" do
         asset.upload_success!
 
-        expect(File.exist?(File.dirname(path))).to be_falsey
```

Comment:
hmm did it stop working?

Reply:
Yeah some new rubocop rules were introduced in this repo but somehow this one was missed. Andrew said just to put it in this PR too :)